content/docs/searching.md

```diff
@@ -128,6 +129,16 @@ If a search action is invoked by keyboard command with the Find dialog window op
 
 *Disclaimer:*  It is Notepad++'s design intention to fulfill some basic searching/replacing capability.  As such, Notepad++ searching is not infinitely flexible and capable of meeting all needs.  For such power needs, please turn to external tools, some of which integrate well with Notepad++.  Integrating well means that after such tools produce results, they can tell Notepad++ which files to open and which line and column numbers to move the caret to, in order to work with matched results.  Examples of such power file/text searching tools might be:  "GrepWin", "PowerGREP", "FileSeek", "Everything" and many others.
 
+#### Wrap Around
+
+- When executing a **Find Next** or a single-step **Replace**, the document text is examined for a match starting at the caret position and proceeding forward toward the end-of-file; if the search text is not found and **☐ Wrap around** is not checkmarked, the search reports that it can't find the text and stops.  However, if **☑ Wrap around** is checkmarked in this situation, searching begins again from the start-of-file (i.e., it wraps around) and text is examined until the caret position is reached (or a search hit occurs).
+
+- When executing a **Find Previous** or **▲** (or a **Find Next** with **☑ Backward direction** checkmarked) or a single-step _Replace_ (with **☑ Backward direction** checkmarked), the document text is examined for a match starting at the caret position and proceeding backward toward the start-of-file; if the search text is not found and **☐ Wrap around** is not checkmarked, the search reports that it can't find the text and stops.  However, if **☑ Wrap around** is checkmarked in this situation, searching begins again from the end-of-file (i.e., it wraps around) and text is examined until the caret position is reached (or a search hit occurs).
```

Comment:
There really is no "Find Previous" text in the UI, so I don't know if you want to leave this looking like there is?  (There's only the little "up" symbol in the UI)

Reply:
fixed